kasa/iot/iotmodule.py

```diff
@@ -24,16 +27,16 @@ def _merge_dict(dest: dict, source: dict) -> dict:
 class IotModule(Module):
     """Base class implemention for all IOT modules."""
 
-    def call(self, method, params=None):
+    async def call(self, method: str, params: dict | None = None) -> dict:
         """Call the given method with the given parameters."""
-        return self._device._query_helper(self._module, method, params)
+        return await self._device._query_helper(self._module, method, params)
```

Comment:
Changed to be consistent with smartmodule. Perhaps we should define to be a part of the common interface in `Module`?